packages/core/app/components/utils/useValues.js

```diff
@@ -143,6 +143,26 @@ const cleanValues = (object, reference) => {
   return object;
 };
 
+/**
+ * Turns the input object into a hash that can be used as
+ * a name for storing the latest values in localStorage.
+ *
+ * @param{object} inputs
+ * @returns{string}
+ */
+const generateHashFromInputsObject = inputs => {
+  const inputsAsString = stringify(inputs);
+
+  let hash = 0;
+  if (inputsAsString.length === 0) return hash;
+  for (let i = 0; i < inputsAsString.length; i++) {
+    const char = inputsAsString.charCodeAt(i);
+    hash = (hash << 5) - hash + char;
+    hash |= 0;
+  }
+  return hash.toString();
+};
+
```

Comment:
I'm not smart enough to figure out if this can result in a super long has if the inputs are long? Should we use something that creates short hashes of e.g. 10-12 characters?

Reply:
The bitshift in line 160 should take care of the value never getting too long as any bits shifted of are discarded. But as this is also very old stuff from Uni for I added a substring to the end to be safe 😊